sycl/include/sycl/ext/intel/experimental/esimd/simd.hpp

```diff
@@ -30,14 +30,18 @@ namespace intel {
 namespace experimental {
 namespace esimd {
 
-/// The simd vector class.
+/// @{
+/// @ingroup sycl_esimd_core
+
+/// The main simd vector class.
 ///
-/// This is a wrapper class for llvm vector values. Additionally this class
+/// A vector of elements, which compiler tries to map to a GPU register.
+/// Supports all standard C++ unary and banary
```

Comment:
```suggestion
/// Supports all standard C++ unary and binary
```
It looks like this sentence is not finished.

Reply:
Thanks for the catch!